service/src/test/java/greencity/security/service/OwnSecurityServiceImplTest.java

```diff
@@ -121,18 +120,19 @@ public void init() {
             .userStatus(UserStatus.ACTIVATED)
             .build();
     }
-/*
+
     @Test
     void signUp() {
-        User user = User.builder().verifyEmail(new VerifyEmail()).build();
-        UserVO userVO = UserVO.builder().verifyEmail(new VerifyEmailVO()).build();
+        User user = User.builder()
+                .id(1L)
+                .verifyEmail(new VerifyEmail()).build();
+        UserVO userVO = UserVO.builder()
+                .id(1L)
+                .verifyEmail(new VerifyEmailVO()).build();
```

Comment:
Please, move this to ModelUtils class. It is also better to use Random class to generate numeric values

Reply:
Done